src/rpc/server.ts

```diff
@@ -865,4 +865,14 @@ export class Server {
   public async getFeeStats(): Promise<Api.GetFeeStatsResponse> {
     return jsonrpc.postObject(this.serverURL.toString(), 'getFeeStats');
   }
+
+  /**
+   *  Provides information about the current version details of the Soroban RPC and captive-core
+   *  @returns {Promise<Api.GetVersionInfoResponse>} the version info
+   *  @see https://developers.stellar.org/docs/data/rpc/api-reference/methods/getVersionInfo
+   * */
```

Comment:
nit: extra spaces, let's match the others

```suggestion
  /**
   * Provides information about the current version details of the Soroban RPC and captive-core
   * @returns {Promise<Api.GetVersionInfoResponse>} the version info
   * @see https://developers.stellar.org/docs/data/rpc/api-reference/methods/getVersionInfo
   */
```